docker-compose.example.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  warframestatus:
+    # Or . if you wanna build the image from source
+    image: ghcr.io/WFCD/warframestatus:latest
```

Comment:
i don't think we publish them at all right now. i'm down for publishing to github, but i haven't done that with any of the dockerized projects as of yet

Reply:
Should build from source now